processor/parse/gemoji-parser.js

```diff
@@ -1,4 +1,4 @@
-const Emoji = require('../../lib/gemoji');
```

Comment:
where did the name `gemoji` come from?? 🤔

Reply:
https://github.com/github/gemoji

Reply:
I was so dissappointed when I learned that it's just **G** (ithub) + **emoji**